components/brave_sync/brave_sync_service_unittest.cc

```diff
@@ -887,3 +886,55 @@ TEST_F(BraveSyncServiceTest, ExponentialResend) {
   EXPECT_EQ(brave_sync_prefs()->GetRecordToResendMeta(record_a_object_id),
             nullptr);
 }
+
+TEST_F(BraveSyncServiceTest, GetDevicesWithFetchSyncRecords) {
+  using brave_sync::jslib_const::kPreferences;
+
+  // Expecting SendFetchSyncRecords will be invoked, not SendFetchSyncDevices
```

Comment:
We don't have to mentioned `SendFetchSyncDevices` here since it is already removed